rbc/tests/test_omnisci_array.py

```diff
@@ -404,13 +405,18 @@ def fn_issue197(x):
 
     omnisci(f'{typ}[]({typ}[])')(fn_issue197)
 
-    _, result = omnisci.sql_execute(
-        f'SELECT {col}, {fn_name}({col}) FROM {omnisci.table_name};'
-    )
+    if 'trunc' in suffix:
+        with pytest.raises(TypingError, match='mismatching types'):
+            omnisci.register()
```

Comment:
What is going on here? Does this indicate that something is not cleaned up properly from the previous tests?